src/api/spec/helpers/validation_helper_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe ValidationHelper do
+  describe '#valid_project_name?' do
+    it 'calls Project.valid_name?' do
+      expect(Project).to receive(:valid_name?).with("home:mschnitzer")
+      valid_project_name?("home:mschnitzer")
+    end
+  end
+
+  describe '#valid_project_name!' do
+    it 'raises an exception if a project name is invalid' do
+      expect { valid_project_name!("home mschnitzer") }.to raise_error(ValidationHelper::InvalidProjectNameError)
```

Comment:
of course that it makes sense, if you are only testing that it raises an exception with `home mschnitzer`, not when name is invalid.

And yesterday you liked the idea... you removed the comment... :unamused: 

Anyway, I am okey with merging this. I can add property tests afterwards if you don't want to do it.